### DISCUSSION TITLE
Historische Karten

### DISCUSSION THREAD

**Discussion by SamLowrie** [copystring/ioBroker.roborock#164] [Q&A]
Ich hätte gerne die historischen Karten passend zu den cleaningInfo.Records.X. Ist es eine gute Idee ein onChange Event auf "Records.0.End" zu legen und dann die Karte aus map direkt als (neuen) Datenpunkt in den Record.0.map zu legen? Oder kommt dann der Adapter durcheinander (z.B. beim rotieren/beim endültigen löschen)? 

Interessant wäre evtl. auch das Optional in den Einstellungen auswählen zu können.

**Comment by copystring** [comment]
Ja, ich wollte mich schon mal darum kümmern, hatte es aber dann aus den Augen verloren.
Ich werde mich demnächst darum kümmern.

**Comment by copystring** [comment]
Wird noch ein bisschen dauern. Den Befehl habe ich gefunden, aber der Server von Roborock antwortet darauf nicht. Dauert also etwas länger.

**Comment by copystring** [comment]
Ich konnte das endlich einbauen. Dafür musste ich erstmal eine passende Möglichkeit aufbauen, den Datenverkehr der Handyapp mitzuschneiden und sauber zu entschlüsseln. Siehe hier: https://forum.iobroker.net/topic/62328/roborock-adapter-tester-gesucht/743?_=1688416908639
Siehe hier: https://github.com/copystring/ioBroker.roborock/commit/6ffdb0e809c68c99e1c6a1c228f8e54d91a05a49

Die Karten kannst du dann jeweils hier finden:

![image](https://github.com/copystring/ioBroker.roborock/assets/1298861/d56f9b8d-0aa1-4259-bdc3-448fc0cab6e5)
